var/spack/repos/builtin/packages/hadoop/package.py

```diff
@@ -0,0 +1,23 @@
+from spack import *
+
+import distutils
+from distutils import dir_util
+
+
+class Hadoop(Package):
+    """
+    The Apache Hadoop software library is a framework that
+    allows for the distributed processing of large data sets
+    across clusters of computers using simple programming models.
+    """
```

Comment:
The error message from the documentation tests isn't very clear, but basically, we run `spack package-list` to create [this beautiful page](http://spack.readthedocs.io/en/latest/package_list.html). `spack package-list` looks at the docstrings of every package to print out `package_list.rst`, but if the docstring starts with a newline, it can mess up the formatting. All you need to do is change the docstring to look like:
```python
"""The Apache Hadoop software library is a framework that
allows for the distributed processing of large data sets
across clusters of computers using simple programming models.
"""
```